.cirrus.yml

```diff
@@ -574,7 +574,7 @@ local_integration_test_task: &local_integration_test_task
         - unit_test
     matrix: *platform_axis
     gce_instance: *standardvm
-    timeout_in: 90m
+    timeout_in: 50m
```

Comment:
@cevich @edsantiago Please let me know if the timeout change is ok for you.

Reason: Tests do not run that long and if there is a new bug which causes a hang we just wait for no good reason.

Reply:
I agree that 90m is probably too long.  But 50 seems a little short to me.  I remember not long ago, e2e tests taking 45min (wrong?).  There could easily be networking hiccups/slowdowns, and the set of tests is always expanding.  Could we do 55m as a compromise?

Reply:
I had 45 min for all the times I pushed this PR without issues, so I added a bit more margin to 50 mins. But lets wait for @edsantiago because he has the timing script and should know better what is realistic.